infrastructure/main.tf

```diff
@@ -7,7 +7,38 @@ resource "azurerm_resource_group" "rg" {
 }
 
 locals {
-  ase_name = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+  ase_name               = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+  is_preview             = "${(var.env == "preview" || var.env == "spreview")}"
+  previewVaultName       = "${var.product}-bsp"
```

Comment:
could this have a `-preview` prefix for clarity?

Reply:
I think it's following the current bulk scan processing setup - config doesn't have `-preview` suffix.

Reply:
As for `-preview` suffix, the name has to be the name of the existing vault (shared with bulk-scan-processor), so we can't change it just here. Yes, there's a plan to add common infrastructure (I'll add a story for it), but it's not a part of this one.